app/lib/constants/site_config.rb

```diff
@@ -215,6 +215,14 @@ module SiteConfig
         description: "Determines the mininum for the periodic email digest",
         placeholder: 2
       },
+      recaptcha_site_key: {
+        description: "Site key for Google ReCAPTCHA, used for reporting abuse.",
```

Comment:
_Small, absolutely non-blocking nitpick_: I think it would be nice to, anywhere the "r" in reCAPTCHA is capitalized, change it to be lowercase to better follow the capitalization in Google's reCAPTCHA docs and our own internal docs that you wrote. 😊  
```suggestion
        description: "Site key for Google reCAPTCHA, used for reporting abuse.",
```